content/how-to/cloud-servers/import-database-to-a-local-mysql-instance/index.md

```diff
@@ -0,0 +1,68 @@
+---
+permalink:/import-database-to-a-local-mysql-instance
```

Comment:
The checks are failing. Maybe add a space before the / ?

Reply:
I have made the change, may you please check?